plugin/src/ChangeBatcher/encodePatchUpdate.lua

```diff
@@ -7,28 +7,33 @@ return function(instance, instanceId, properties)
 	local update = {
 		id = instanceId,
 		changedProperties = {},
+		requiresRecreate = false
 	}
 
 	for propertyName in pairs(properties) do
 		if propertyName == "Name" then
 			update.changedName = instance.Name
-		else
```

Comment:
This change, whilst structural (it now encodes the **Name** field as well as setting the **changedName** property) has not changed the intended logic and is more concise.